CriticalMass/SettingsViewController.swift

```diff
@@ -52,28 +52,18 @@ class SettingsViewController: UITableViewController {
         }
     }
 
-    // MARK: Actions
-
-    @IBAction func gpsCellAction(_: UISwitch) {}
```

Comment:
I was actually a fan of having the `@IBAction` in the controller. Makes it easy to understand. Where is the switch target now?

Reply:
The cell no gets a switchable element that can handle the update